lib/src/widgets/message_view.dart

```diff
@@ -106,63 +104,69 @@ class _MessageViewState extends State<MessageView>
             child: Padding(
               padding: EdgeInsets.only(
                   bottom: widget.message.reaction.reactions.isNotEmpty ? 6 : 0),
-              child: message.isAllEmoji
-                  ? Stack(
-                      clipBehavior: Clip.none,
-                      children: [
-                        Padding(
-                          padding: emojiMessageConfiguration?.padding ??
-                              EdgeInsets.fromLTRB(
-                                leftPadding2,
-                                4,
-                                leftPadding2,
-                                widget.message.reaction.reactions.isNotEmpty
-                                    ? 14
-                                    : 0,
-                              ),
-                          child: Transform.scale(
-                            scale: widget.shouldHighlight
-                                ? widget.highlightScale
-                                : 1.0,
-                            child: Text(
-                              message,
-                              style: emojiMessageConfiguration?.textStyle ??
-                                  const TextStyle(fontSize: 30),
+              child: (() {
```

Comment:
why a widget parameter is taking anonymous function?

Reply:
We have use case where we need to show widgets according to conditions, We can use anonymous functions other wise we need to use ternory condition.